packages/qwik/src/optimizer/src/plugins/plugin.ts

```diff
@@ -574,9 +574,9 @@ export function createPlugin(optimizerOptions: OptimizerOptions = {}) {
       debug(`load()`, 'Found', id);
       let { code } = transformedModule[0];
       const { map, segment } = transformedModule[0];
+      const firstInput = Object.values(opts.input || [])[0];
```

Comment:
Same. Are these the only two places checking opts.input? I thought there were more